velox/external/duckdb/CMakeLists.txt

```diff
@@ -13,7 +13,9 @@ add_subdirectory(tpch)
 
 add_compile_definitions(DISABLE_DUCKDB_REMOTE_INSTALL)
 add_compile_definitions(BUILD_PARQUET_EXTENSION)
-
+if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
It would be nice to add a comment to explain this check.

Reply:
will do.